apps/store/themes/store/js/asset.js

```diff
@@ -1,13 +1,24 @@
 $(function () {
-    $('#btn-add-gadget').click(function () {
-        var elem = $(this);
-        asset.process(elem.data('type'), elem.data('aid'), location.href, elem);
-    });
+    $('.action-container').on(
+        {
+            mouseenter: function () {
+                $(this).find("i").removeClass().addClass("fa fa-remove");
+            },
+            mouseleave: function () {
+                $(this).find("i").removeClass().addClass("fa fa-star");
+            },
+            click: function () {
+                var elem = $(this);
+                asset.unsubscribeBookmark(elem.data('type'), elem.data('aid'), location.href, elem);
+            }
+        },'#btn-remove-subscribe').on(
+        {
```

Comment:
Thanks @splinter , I have fixed in in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/fcdafbb8459a7abb3f87114b021f2b565fd85bfc)